### ISSUE TITLE
Logout link - Chrome browser is not keyboard accessible

### ISSUE THREAD

**Issue by JohnHewittCriterion** [flexion/fs-intake-module#830]
Steps To Verify With JAWS:
 
Launch the website in the Chrome browser.
Activate the "Temp outfitter's permitLogin to apply now with login.gov logo" link.
Open the links list box with the key combination Insert + F7. Review to find the "Logout" link.
Move to the link on the page. JAWS announces 'Link Logout'.
All functionality of the content must be operable through a keyboard interface.

The "Logout" link is present in both Chrome and Firefox. User can reach the Logout link from the links list box. User cannot reach the link pressing the Tab key.
In IE the "Logout" is present but not recognized as a link.

WCAG: 2.1 Keyboard Accessible - 2.1.1 
EPERM-136

**Comment by sethalt** [comment]
- [ ] Check with Mac Voice Over
- [x] Check with JAWS
